CHANGELOG.md

```diff
@@ -32,12 +32,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * **BREAKING** Renamed `dst` to `dst_registry` in `crane.push` function.
 * **BREAKING** Renamed `image_path` to `tar_path` keyword argument to aligne with kaniko module.
 * **BREAKING** Renamed `dst` and `src` to `dst_registry` and `src_registry`.
-* **BREAKING** `dive`: Replace "/" with "_" in image_name. Image names contains namespaces which are separated by "/" to ensure image name is a file instead of a directory structure.
-* **BREAKING** `trivy`: Replace "/" with "_" in image_name. Image names contains namespaces which are separated by "/" to ensure image name is a file instead of a directory structure.
+* **BREAKING** `dive`: Replace "/" with "_" in image_name. Image names contains stages which are separated by "/" to ensure image name is a file instead of a directory structure.
```

Comment:
Please do not touch old CHANGELOG entries.

Reply:
fixed